akka-cluster/src/main/scala/akka/cluster/Cluster.scala

```diff
@@ -411,8 +427,36 @@ class Cluster(val system: ExtendedActorSystem) extends Extension {
 
   private def closeScheduler(): Unit = scheduler match {
     case x: Closeable ⇒ x.close()
-    case _            ⇒
+    case _            ⇒ // ignore, this is fine
+  }
+
+  // ========================================================
+  // ================== OLDEST MAINTANANCE ==================
+  // ========================================================
+
+  /** INTERNAL API: Returns state.members sorted by Member.ageOrdering (from oldest to youngest) */
+  @InternalApi private[cluster] def membersByAge(): immutable.SortedSet[Member] =
+    SortedSet.empty(Member.ageOrdering) ++ state.members.toList // FIXME
```

Comment:
note to self, `union` and drop the toList, was checking if ordering actually worked as I expected